scripts/install-chronograf.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Check If Chronograf Has Been Installed
+
+if [ -f /home/vagrant/.chronograf ]
+then
+    echo "Chronograf already installed."
+    exit 0
+fi
+
+touch /home/vagrant/.chronograf
+
+wget -q https://dl.influxdata.com/chronograf/releases/chronograf_1.5.0.1_amd64.deb
+sudo dpkg -i chronograf_1.5.0.1_amd64.deb
+
```

Comment:
Do we need to `rm chronograf_1.5.0.1_amd64.deb`? We wouldn't want the script to leave the artifact.

Reply:
And maybe move to it `/tmp/ instead of current directory.